### PR TITLE
add wand of atonement, change rod lores

### PR DESCRIPTION
**Pull request by fireflame606**
hello mr moon

### CODE REVIEW

**Comment by DeDiamondPro** [comment]
Why did you create a new pr, just add it to your other one next time

**Comment by fireflame606** [comment]
too many rods in this game ill do some more later

File: items/FARMER_ROD.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
 {
   "itemid": "minecraft:fishing_rod",
   "displayname": "§aFarmer\u0027s Rod",
-  "nbttag": "{Unbreakable:1,HideFlags:254,display:{Lore:[0:\"§7Damage: §c+45\",1:\"\",2:\"§7Chance to fish up farm animals!\",3:\"\",4:\"§7Increases fishing speed by §940%\",5:\"\",6:\"§7§8This item can be reforged!\",7:\"§4❣ §cRequires §aFishing Skill 8\",8:\"§a§lUNCOMMON FISHING ROD\"],Name:\"§aFarmer\u0027s Rod\"},ExtraAttributes:{id:\"FARMER_ROD\"}}",
+  "nbttag": "{Unbreakable:1,HideFlags:254,display:{Lore:[0:\"§7Damage: §c+50\",1:\"§7Strength: §c+20\",2:\"\",3:\"§7Chance to fish up farm animals!\",4:\"\",5:\"§7Increases fishing speed by §940%\",6:\"\",7:\"§7§8This item can be reforged!\",8:\"§4❣ §cRequires §aFishing Skill 8\",9:\"§a§lUNCOMMON FISHING ROD\"],Name:\"§aFarmer\u0027s Rod\"},ExtraAttributes:{id:\"FARMER_ROD\"}}",
+  "modver": "2.0.0-REL",
```
Comment (IRONM00N) [inline items/FARMER_ROD.json:5]: Why are there 2 of these?

**Review by IRONM00N — CHANGES_REQUESTED** [review]
Looks good other than the duplicate mod version

**Pull request closed without merging**